.github/workflows/renovate.yml

```
@@ -0,0 +1,31 @@
+name: Renovate
+on:
+  # Allows manual/automated trigger for debugging purposes
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: "Override default log level"
+        required: false
+        default: "info"
+        type: string
+      overrideSchedule:
+        description: "Override all schedules"
+        required: false
+        default: "false"
+        type: string
+  schedule:
+    - cron: '0 8 * * *'
+jobs:
+  renovate:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4.1.1
+      # Don't waste time starting Renovate if JSON is invalid
+      - name: Validate Renovate JSON
+        run:  npx --yes --package renovate -- renovate-config-validator
+      - name: Self-hosted Renovate
+        uses: renovatebot/github-action@v40.1.5
```

Found that also this is actually needed, otherwise I got `No repositories found - did you want to run with flag --autodiscover?`:
```suggestion
        uses: renovatebot/github-action@v40.1.5
        env:
          RENOVATE_REPOSITORIES: ${{ github.repository }}
```

---

I thought that renovate does this by default hence didn't include it. Thanks for the correction!

---

Yeah, not sure why that's needed either, but running it on my fork gave me that error 🤔